terraform/workspaces/paragon/helm/helm.tf

```diff
@@ -130,14 +122,25 @@ resource "helm_release" "metricsserver" {
   ]
 }
 
+######
+# Microservices
+######
+
+# helm hash
+module "helm_hash_onprem" {
+  source = "../helm-hash"
+  chart_directory = "./charts/paragon-onprem"
+}
+
 # microservices deployment
 resource "helm_release" "paragon_on_prem" {
   name             = "paragon-on-prem"
   description      = "Paragon microservices"
   chart            = "./charts/paragon-onprem"
-  namespace        = "paragon"
-  cleanup_on_fail  = true
+  version          = "${var.helm_values.global.env["VERSION"]}-${module.helm_hash_onprem.hash}"
```

Comment:
[nit] Could we make this non-sensitive so we can see the hash? I couldn't see it when I tried to run an apply
```suggestion
  version          = "${nonsensitive(var.helm_values.global.env["VERSION"])}-${module.helm_hash_onprem.hash}"
```

Reply:
Hmm.. I see it here:
![image](https://github.com/useparagon/aws-on-prem/assets/114088924/0bb94cf1-36e1-4005-9ab4-7912b66b70bd)

Could it be something else that was marked sensitive?